libexec/scoop-cache.ps1

```diff
@@ -28,7 +28,11 @@ function cacheshow($app) {
 
     $files | ForEach-Object { cacheinfo $_ } | Select-Object Name, Version, Length, URL
 
-    Write-Host "Total: $($files.Length) $(pluralize $files.Length 'file' 'files'), $(filesize $totalLength)" -ForegroundColor Yellow
+    if (!$files.Length) {
+        Write-Host -ForegroundColor Yellow "No cache found."
```

Comment:
I think this message is slightly misleading. It means that nothing related to cache exists. While in fact the ~/scoop/cache folder will always exist. 

The previous message is clear enough IMO and is consistent with other output messages in `cache` command.

Reply:
The problem is, when there is nothing in `$SCOOP\cache`, `scoop cache show` will show this:

```
Total: 0 files,  B
```

since `$totalLength` has no value.

So should we set `$totalLength = 0` in this case? It will show:

```
Total: 0 files, 0 B
```

which is fair enough.

Reply:
Yes, we can do that

Reply:
Great. I have made the change.